### PR TITLE
Update URLs in documentation

### LINKED ISSUE #67357: Fix batch of broken links in module docs
We're trying to fix as many broken links as possible before modules move into collections.  This is the batch of broken links on some Ansible modules.

NOTE: the link checker sometimes reports an error where a link actually works. Ignore those if you find them.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME

docs.ansible.com

###### BROKEN LINKS

~~https://docs.ansible.com/ansible/latest/modules/openssl_privatekey_module.html#openssl-privatekey-module~~
~~├─BROKEN─ https://en.wikipedia.org/wiki/RSA_(cryptosystem~~

~~https://docs.ansible.com/ansible/devel/modules/elb_target_info_module.html#elb-target-info-module~~
~~└─BROKEN─ https://boto3.readthedocs.io/en/latest/%20reference/services/elbv2.html#ElasticLoadBalancingv2.Client.describe_target_health~~

~~https://docs.ansible.com/ansible/devel/plugins/lookup/laps_password.html~~
~~├─BROKEN─ https://keathmilligan.net/python-ldap-and-macos/~~

~~https://docs.ansible.com/ansible/devel/modules/acme_certificate_revoke_module.html#acme-certificate-revoke-module~~
~~─BROKEN─ http://0.0.0.0:1337/modules/Section%205.3.1%20of%20RFC5280~~

https://docs.ansible.com/ansible/devel/modules/airbrake_deployment_module.html#airbrake-deployment-module
└─BROKEN─ http://help.airbrake.io/kb/api-2/deploy-tracking 

https://docs.ansible.com/ansible/devel/modules/consul_module.html#consul-module
├─BROKEN─ http://0.0.0.0:1337/v1/agent/service/register 

~~https://docs.ansible.com/ansible/devel/modules/hetzner_firewall_module.html#hetzner-firewall-module~~
~~─BROKEN─ http://0.0.0.0:1337/modules/the%20documentation,https:/wiki.hetzner.de/index.php/Robot_Firewall/en#Parameter~~

~~https://docs.ansible.com/ansible/devel/modules/keycloak_client_module.html#keycloak-client-module~~
~~├─BROKEN─ http://www.keycloak.org/docs-api/3.3/rest-api/~~
~~├─BROKEN─ http://www.keycloak.org/docs-api/3.3/rest-api/index.html#_resourceserverrepresentation~~

~~https://docs.ansible.com/ansible/devel/modules/keycloak_clienttemplate_module.html#keycloak-clienttemplate-module~~
~~├─BROKEN─ http://www.keycloak.org/docs-api/3.3/rest-api/~~ 

~~https://docs.ansible.com/ansible/devel/modules/keycloak_group_module.html#keycloak-group-module~~
~~├─BROKEN─ http://www.keycloak.org/docs-api/3.3/rest-api/~~

https://docs.ansible.com/ansible/devel/modules/meraki_network_module.html#meraki-network-module
├─BROKEN─ http://0.0.0.0:1337/modules/my.meraki.com 

~~https://docs.ansible.com/ansible/devel/modules/os_ironic_module.html#os-ironic-module~~
~~├─BROKEN─ https://docs.openstack.org/ironic/latest/install/include/root-device-hints.html~~ 

~~https://docs.ansible.com/ansible/devel/modules/ovh_ip_failover_module.html#ovh-ip-failover-module~~
~~├─BROKEN─ https://eu.api.ovh.com/g934.first_step_with_api~~ 

~~https://docs.ansible.com/ansible/devel/modules/ovh_ip_loadbalancing_backend_module.html#ovh-ip-loadbalancing-backend-module~~
~~├─BROKEN─ https://eu.api.ovh.com/g934.first_step_with_api~~

~~https://docs.ansible.com/ansible/devel/modules/packet_volume_attachment_module.html#packet-volume-attachment-module~~
~~├─BROKEN─ https://www.packet.net/developers/api/volumeattachments/~~

https://docs.ansible.com/ansible/devel/modules/postgresql_info_module.html#postgresql-info-module
├─BROKEN─ https://www.postgresql.org/docs/current/catalog-pg-replication-slots.html

~~https://docs.ansible.com/ansible/devel/modules/postgresql_table_module.html#postgresql-table-module~~
~~├─BROKEN─ http://0.0.0.0:1337/modules/postgresql.org/docs/current/datatype.html~~ 

https://docs.ansible.com/ansible/devel/modules/win_credential_module.html#win-credential-module
├─BROKEN─ https://docs.microsoft.com/en-us/windows/desktop/api/wincred/ns-wincred-_credentiala 

https://docs.ansible.com/ansible/devel/modules/win_dsc_module.html#win-dsc-module
├─BROKEN─ https://docs.microsoft.com/en-us/powershell/dsc/resources/resources 

https://docs.ansible.com/ansible/devel/modules/win_inet_proxy_module.html#win-inet-proxy-module
├─BROKEN─ http://0.0.0.0:1337/modules/host 

https://docs.ansible.com/ansible/devel/modules/win_webpicmd_module.html#win-webpicmd-module
└─BROKEN─ http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release 

~~https://docs.ansible.com/ansible/devel/modules/xenserver_guest_module.html#xenserver-guest-module~~
~~├─BROKEN─ https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI.py~~ 

~~https://docs.ansible.com/ansible/devel/modules/xenserver_guest_powerstate_module.html#xenserver-guest-powerstate-module~~
~~├─BROKEN─ https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI.py~~ 

~~https://docs.ansible.com/ansible/devel/modules/xenserver_guest_info_module.html#xenserver-guest-info-module~~
~~├─BROKEN─ https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI.py~~

https://docs.ansible.com/ansible/devel/modules/zabbix_map_module.html#zabbix-map-module
├─BROKEN─ https://en.wikipedia.org/wiki/DOT_(graph_description_language

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Partially fixes: #67357

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE 
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/amazon/elb_target_info.py
lib/ansible/modules/cloud/ovh/ovh_ip_failover.py
lib/ansible/modules/cloud/ovh/ovh_ip_loadbalancing_backend.py
lib/ansible/modules/cloud/packet/packet_volume_attachment.py
lib/ansible/modules/cloud/xenserver/xenserver_guest.py
lib/ansible/modules/cloud/xenserver/xenserver_guest_info.py
lib/ansible/modules/cloud/xenserver/xenserver_guest_powerstate.py
lib/ansible/modules/identity/keycloak/keycloak_client.py
lib/ansible/modules/identity/keycloak/keycloak_clienttemplate.py
lib/ansible/modules/identity/keycloak/keycloak_group.py
lib/ansible/plugins/lookup/laps_password.py

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/elb_target_info.py [hunk 1]
```diff
@@ -179,10 +179,10 @@
                     sample:
                         - us-west-2a
                 target_health:
-                    description: the target health description
-                                 (see U(https://boto3.readthedocs.io/en/latest/
-                                  reference/services/elbv2.html#ElasticLoadBalancingv2.Client.describe_target_health))
-                                 for all possible values
+                    description:
+                    - The target health description
```
Comment (tremble) [inline lib/ansible/modules/cloud/amazon/elb_target_info.py]:
```suggestion
                    - The target health description.
```

File: lib/ansible/modules/cloud/amazon/elb_target_info.py [hunk 2]
```diff
@@ -179,10 +179,10 @@
...
+                    - See following link for all the possible values
+                    - U(https://boto3.readthedocs.io/en/latest/reference/services/elbv2.html#ElasticLoadBalancingv2.Client.describe_target_health)
```
Comment (tremble) [inline lib/ansible/modules/cloud/amazon/elb_target_info.py]:
```suggestion
                      U(https://boto3.readthedocs.io/en/latest/reference/services/elbv2.html#ElasticLoadBalancingv2.Client.describe_target_health)
```

**Review by tremble — CHANGES_REQUESTED** [review]

**Review by tremble — APPROVED** [review]
LGTM

New URLs all resolve and look like the right things at first glance.

**Comment by ndclt** [comment]
Looks good for the Keycloak urls. Thanks.

File: lib/ansible/modules/cloud/xenserver/xenserver_guest.py [hunk 3]
```diff
@@ -28,7 +28,7 @@
 - 'To acquire XenAPI Python library, just run C(pip install XenAPI) on your Ansible Control Node. The library can also be found inside
    Citrix Hypervisor/XenServer SDK (downloadable from Citrix website). Copy the XenAPI.py file from the SDK to your Python site-packages on your
    Ansible Control Node to use it. Latest version of the library can also be acquired from GitHub:
-   https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI.py'
+   https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI/XenAPI.py'
```
Comment (felixfontein) [inline lib/ansible/modules/cloud/xenserver/xenserver_guest.py]:
```suggestion
   U(https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI/XenAPI.py)'
```
(Same for the ones below.)

Comment (bvitnik) [reply lib/ansible/modules/cloud/xenserver/xenserver_guest.py]: In the docs I see that URLs are already automatically detected and converted to hyperlinks. I'm not sure if `U()` makes any difference.

Comment (felixfontein) [reply lib/ansible/modules/cloud/xenserver/xenserver_guest.py]: IMO it's bad style to rely on implicit undocumented conversions where there is an explicit way of doing it.

**Review by felixfontein — APPROVED** [review]
shipit

**Pull request merged**